RecoLocalMuon/Configuration/python/RecoLocalMuon_EventContent_cff.py

```diff
@@ -28,13 +28,11 @@
         'keep *_cscSegments_*_*', 
         'keep *_rpcRecHits_*_*')
 )
-
-def _modifyRecoLocalMuonEventContentForPhase2( object ):
-    object.outputCommands.append('keep *_gemRecHits_*_*')
-    object.outputCommands.append('keep *_me0RecHits_*_*')
-    object.outputCommands.append('keep *_me0Segments_*_*')
-
 from Configuration.StandardSequences.Eras import eras
-eras.phase2_muon.toModify( RecoLocalMuonFEVT, func=_modifyRecoLocalMuonEventContentForPhase2 )
-eras.phase2_muon.toModify( RecoLocalMuonRECO, func=_modifyRecoLocalMuonEventContentForPhase2 )
-eras.phase2_muon.toModify( RecoLocalMuonAOD,  func=_modifyRecoLocalMuonEventContentForPhase2 )
+def _updateOutput( era, outputPSets, commands):
```

Comment:
for the next PR... this should just get moved to normal cff that manage the event content. If the collections are not produced, they won't be part of the event content
@dildick